ambry-utils/src/test/java/com.github.ambry.utils/UtilsTest.java

```diff
@@ -130,6 +148,19 @@ public void testReadBuffers()
     for (int i = 0; i < 40000; i++) {
       Assert.assertEquals(buf[i + 4], outputBuf.array()[i]);
     }
+    // 0 size
+    inputBuf.rewind();
+    inputBuf.putInt(0, 0);
```

Comment:
If i don't specify the index, I need to rewind again since the position will be incremented.